components/common/ConnectWallet/ConnectWalletModal.vue

```diff
@@ -91,8 +91,12 @@ const { $i18n } = useNuxtApp()
 const selectedWalletProvider = ref<BaseDotsamaWallet>()
 const hasSelectedWalletProvider = ref(false)
 const forceWalletSelect = ref(false)
+const { $store } = useNuxtApp()
 const identityStore = useIdentityStore()
 
+// urlPrefix from usePrefix() would not update inside modal component
+const urlPrefix = computed(() => $store.getters.currentUrlPrefix)
```

Comment:
hmmm... this is strange prefix should be based on this value too

Reply:
#5984, time to update?

Reply:
Not sure why urlPrefix does not update using `usePrefix`in the modal component. Maybe leave it to https://github.com/kodadot/nft-gallery/issues/5984 for deeper invigorate.